prowler/providers/aws/services/elasticache/elasticache_service.py

```diff
@@ -32,7 +32,9 @@ def __describe_cache_clusters__(self, regional_client):
                         id=cache_cluster["CacheClusterId"],
                         arn=cluster_arn,
                         region=regional_client.region,
-                        cache_subnet_group_id=cache_cluster["CacheSubnetGroupName"],
+                        cache_subnet_group_id=cache_cluster.get(
```

Comment:
You should set `cache_subnet_group_id: Optional[str]` in the model then.

Reply:
Done